src/keyboard/mod.rs

```diff
@@ -625,3 +657,311 @@ where
         },
     )
 }
+
+/// Implement a keyboard to automatically detect the keymap and send KeyRepeatEvents
+/// at set intervals
+///
+/// This requires you to provide an implementation to receive the events after they
+/// have been interpreted with the keymap, as well as an implementation to be called
+/// when KeyRepeatEvents are sent at intervals set by the KeyRepeatKind argument.
```

Comment:
Please add a note like "This second implementation can be called anytime independently of the dispatching of the wayland events."

Reply:
I've changed it to try and explain it in a way that made sense to me and flowed with what was already there but if you can think of a clearer way to say it then I'll change it to that.